UnicornEngine/WindowManager/include/unicorn/window_manager/Monitor.hpp

```diff
@@ -0,0 +1,148 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#ifndef UNICORN_WINDOW_MANAGER_MONITOR_HPP
+#define UNICORN_WINDOW_MANAGER_MONITOR_HPP
+
+#include <unicorn/window_manager/GammaRamp.hpp>
+#include <unicorn/window_manager/MonitorMemento.hpp>
+#include <unicorn/window_manager/VideoMode.hpp>
+
+#include <unicorn/utility/SharedMacros.hpp>
+
+#include <wink/signal.hpp>
+
+#include <cstdint>
+#include <utility>
+#include <string>
+#include <vector>
+
+namespace unicorn
+{
+namespace WindowManager
+{
+
+/** @brief  Provides information about monitor
+ *
+ *  Proxies calls to window manager adapter
+ */
+class Monitor
+{
+public:
+    //! Alias to MonitorMemento::State
+    typedef MonitorMemento::State State;
```

Comment:
Just not to use MonitorMemento:: everywhere, so that State can be resolved inside Monitor class